package.json

```diff
@@ -16,7 +16,8 @@
     "test-watch": "npm run test -- --watch",
     "lint": "eslint src/ test/",
     "lint-watch": "esw -w src/ test/",
-    "cover": "cross-env NODE_ENV=test nyc npm test"
+    "cover": "cross-env NODE_ENV=test nyc npm test",
+    "postinstall": "npm run build"
```

Comment:
It's an efficiency thing so you don't have to run `npm build` separately after `npm install`, instead it'll run it automatically. You can always run it using just the `npm build` command as well if you choose.